docs/topics/broad-crawls.rst

```diff
@@ -50,23 +50,29 @@ To apply the recommended priority queue use::
 
     SCHEDULER_PRIORITY_QUEUE = 'scrapy.pqueues.DownloaderAwarePriorityQueue'
 
+.. _broad-crawls-concurrency:
+
 Increase concurrency
 ====================
 
 Concurrency is the number of requests that are processed in parallel. There is
-a global limit and a per-domain limit.
+a global limit (:setting:`CONCURRENT_REQUESTS`) and an additional limit that
+can be set either per domain (:setting:`CONCURRENT_REQUESTS_PER_DOMAIN`) or per
+IP (:setting:`CONCURRENT_REQUESTS_PER_IP`).
```

Comment:
May be worth adding a note that CONCURRENT_REQUESTS_PER_IP doesn't work with DownloaderAwarePriorityQueue, which is recommended in a previous section. See also https://github.com/scrapy/scrapy/issues/3867.